src/Board.js

```diff
@@ -0,0 +1,146 @@
+import React, { Component } from 'react'
+import Square from './Square'
+
+let startTime = 0;
+let gameOver = false;
+
+export default class Board extends Component {
+
+  handleClick(i) {
+    if (startTime === 0) {
+      startTime = Date.now();
+    }
+    const squares = this.props.squares.slice();
+    squares[i] = this.props.xIsNext ? '⭐' : '🌞';
+    this.props.setParentsState({
```

Comment:
I see. This is fine, I guess. Normally, it's a bit of an "anti-pattern" to expose state so openly, it kind of ruins the reason to have separate components. But I understand that this guy holds most of the state.